src/libs/ActiveClientManager/index.js

```diff
@@ -24,7 +24,9 @@ function isReady() {
 Onyx.connect({
     key: ONYXKEYS.ACTIVE_CLIENTS,
     callback: (val) => {
-        activeClients = !val ? [] : val;
+        if (val && activeClients.indexOf(val) < 0) {
```

Comment:
Whats this change for? When i test your PR the command `DeleteMembersFromWorkspace` never gets called in the network tab and I think its because of the code here.

Reply:
I had to refactor this a bit because this was using the array merging feature of onyx that we changed. It is not called in the flow of that api command.